src/diffCheck/transformation/DFTransformation.cc

```diff
@@ -0,0 +1,6 @@
+#include "DFTransformation.hh"
+
+namespace diffCheck::transformation
+{
+
```

Comment:
Would You say that it is the proper way to do this, or if the Class has no methods, we should not have any .cc file ?

Reply:
Have a look at upcoming commit from my side for this point.